java.hints/src/org/netbeans/modules/java/hints/suggestions/Lambda.java

```diff
@@ -242,33 +238,27 @@ public static ErrorDescription explicitParameterTypes(HintContext ctx) {
         return ErrorDescriptionFactory.forName(ctx, ctx.getPath(), Bundle.ERR_addExplicitLambdaParameters(), new AddExplicitLambdaParameterTypes(ctx.getInfo(), ctx.getPath()).toEditorFix());
     }
     
-    @Hint(displayName = "#DN_addVarLambdaParameters", description = "#DESC_addVarLambdaParameters", category = "suggestions", hintKind = Hint.Kind.ACTION, minSourceVersion = "11")
-    @Messages({
-        "DN_addVarLambdaParameters=Convert Lambda to Var Parameter Types",
-        "DESC_addVarLambdaParameters=Converts lambdas to var parameter types",
-        "ERR_addVarLambdaParameters=",
-        "FIX_addVarLambdaParameters=Use var parameter types"
-    })
+    @Hint(displayName = "#DN_ConvertVarLambdaParameters", description = "#DESC_ConvertVarLambdaParameters", category = "suggestions", hintKind = Hint.Kind.ACTION, minSourceVersion = "11")
```

Comment:
Generally it is advised to use `@NbBundle.Messages` annotation for localization; the message is then better tied to the source and if the source moves/is refactored, the message is better moved/updated along. 

I suggested to write the description directly to the `Bundle.properties` as it is long and in Java you would need to concatenate Strings, which is (IMHO) less redable than externalizing long multiline string in `Bundle`. 

But both approaches work.